committee.rst

```diff
@@ -117,6 +109,19 @@ These criteria are not exhaustive -- nominations and self-nominations are free
 to strengthen the case in whatever way they deem appropriate --- but a record
 of thoughtful contribution in a public space carries the most weight.
 
+Membership on the committee comes with a three-year term, extended so
+that a term expires only at the end of a nomination process. Call a
+member who has served for over three years an "expiring" member.
+A nomination process is triggered when the number of unexpiring members is
+about to drop below 9.
+
+When a nomination process is triggered, the secretary of
+the committee will put out a public call for nominations for people to join
+the committee. Self-nominations are welcome and expected. Expiring members are
+free to re-nominate themselves. Nominations include a brief
+bio of the nominee and reasons why they would be appropriate for the
+committee.
```

Comment:
Just a reodering (else a reader not interested in self-nominations might stop reading )
```suggestion
When a nomination process is triggered, the secretary of
the committee will put out a public call for nominations for people to join
the committee. Nominations include a brief
bio of the nominee and reasons why they would be appropriate for the
committee. Self-nominations are welcome and expected. Expiring members are
free to re-nominate themselves. 
```